app/cli/internal/action/workflow_run_list.go

```diff
@@ -61,14 +74,19 @@ type PaginationOpts struct {
 
 func (action *WorkflowRunList) Run(opts *WorkflowRunListOpts) (*PaginatedWorkflowRunItem, error) {
 	client := pb.NewWorkflowRunServiceClient(action.cfg.CPConnection)
-	resp, err := client.List(context.Background(),
-		&pb.WorkflowRunServiceListRequest{
-			WorkflowId: opts.WorkflowID,
-			Pagination: &pb.CursorPaginationRequest{
-				Limit:  int32(opts.Pagination.Limit),
-				Cursor: opts.Pagination.NextCursor,
-			},
-		})
+	req := &pb.WorkflowRunServiceListRequest{
+		WorkflowId: opts.WorkflowID,
+		Pagination: &pb.CursorPaginationRequest{
+			Limit:  int32(opts.Pagination.Limit),
+			Cursor: opts.Pagination.NextCursor,
+		},
+	}
+
+	if v, ok := WorkflowRunStatus()[opts.Status]; ok {
```

Comment:
What if `opts.Status` doesn't contain a valid value? (`ok` is false). Should it raise an error?

Reply:
I would check the code in how it's handled in the rest of commands if they are handled and do it in the same way.